docs/index.md

```diff
@@ -7,20 +7,18 @@ Everything you need to know about how to build Apache Airflow® workflows using
 Are you new to DAG Factory? This is the place to start!
 
 * DAG Factory at a glance
-    * [Quickstart with Airflow standalone](getting-started/quick-start-airflow-standalone.md)
-    * [Quickstart with Astro CLI](getting-started/quick-start-astro-cli.md)
+  * [Quickstart with Airflow standalone](getting-started/quick-start-airflow-standalone.md)
+  * [Quickstart with Astro CLI](getting-started/quick-start-astro-cli.md)
 * Install guide
 * [Using YAML instead of Python](./comparison/index.md)
-    * [Traditional Airflow Operators](./comparison/traditional_operators.md)
-    * [TaskFlow API](./comparison/taskflow_api.md)
+  * [Traditional Airflow Operators](./comparison/traditional_operators.md)
+  * [TaskFlow API](./comparison/taskflow_api.md)
 
 ## Features
 
-* Configuring your workflows
-  * Environment variables
-  * Defaults
-* Defining actions upon task states
-  * Callbacks
```

Comment:
Given the work being done in #322 , I feel we could document it better once the PR is merged and hence I am removing this list item.